app/code/Magento/Catalog/view/frontend/templates/product/image_with_borders.phtml

```diff
@@ -12,7 +12,7 @@ $width = (int)$block->getWidth();
 $paddingBottom = $block->getRatio() * 100;
```

Comment:
We don't need these two variables (`$width`, `$paddingBottom`) any more given the other changes in this pull request. (GitHub won't let me comment on line 11, so the suggestion here is incomplete.)
```suggestion
```